src/com/thunder_cut/graphics/controller/DrawingModeHandler.java

```diff
@@ -54,6 +57,12 @@ public DrawingModeHandler() {
             eraser.setSize(eraser.getSize()+2);
             LineDrawer lineDrawer = (LineDrawer) drawingFeatures.get(DrawingMode.LINE);
             lineDrawer.setSize(lineDrawer.getSize() + 2);
+            RectangleDrawer rectDrawer = (RectangleDrawer) drawingFeatures.get(DrawingMode.RECTANGLE);
```

Comment:
We have to clean up and make simple this function.    
The function is grow up size more and more.    
We need method like setBrushSize.